modules/kube-controller-sg/main.tf

```diff
@@ -0,0 +1,48 @@
+/**
+ * ## Kube Controller Security Group
+ *
+ * Define a security group for kube controller
+ *
+ */
+
```

Comment:
you will probably want additional variables for different CIDR blocks. For example, the list of CIDR blocks for SSH might be different for etcd, and the others.

Reply:
@ketzacoatl Then I would add `cidr_blocks_ssh` and `cidr_blocks_etcd` variables on the module. This variable should also be added to `kube-stack-private/variables.tf` with the same default https://github.com/fpco/terraform-aws-foundation/blob/master/examples/kube-stack-private/variables.tf#L43